assets/js/modules/analytics/setup.js

```diff
@@ -674,6 +682,18 @@ class AnalyticsSetup extends Component {
 						</p>
 					</div>
 				}
+				{ onSettingsPage && useSnippet && ! ampClientIDOptIn && ( ! ampEnabled || ( ampEnabled && ampMode !== 'primary' ) ) && (
+					<div className="googlesitekit-setup-module__input">
+						<Switch
+							id="anonymizeIP"
+							label={ __( 'Anonymize IP addresses', 'google-site-kit' ) }
+							onClick={ this.switchStatus( 'anonymizeIP' ) }
+							checked={ anonymizeIP }
+							hideLabel={ false }
+						/>
+						<p>{ anonymizeIP ? __( 'IP addresses will be anonymized.', 'google-site-kit' ) : __( 'IP addresses will not be anonymized.', 'google-site-kit' ) } <Link href="https://support.google.com/analytics/answer/2763052" external inherit dangerouslySetInnerHTML={ { __html: __( 'Learn more<span class="screen-reader-text"> about IP anonymization.</span>', 'google-site-kit' ) } } /></p>
```

Comment:
My other comment is still valid, can you break this into multiple lines? This is way too long. The opening and closing `p` tags could be on their own, then the conditional explanation on a single line, then the link on a single line.

Reply:
Hmm, okay. Going forward we should probably have limits on how long these lines can be; manually line breaks aren't my favourite as they can mess with things on different screen sizes. But as long as this doesn't mess with responsive sizes I'll give that a go.

Reply:
It's a bit odd to break this one as it's much shorter than the AMP opt-in above:

<img width="577" alt="Screenshot 2019-12-13 12 58 44" src="https://user-images.githubusercontent.com/90871/70802067-684fe880-1da8-11ea-9a29-5fac6eda5c4e.png">

and

![Screen Shot 2019-12-13 at 12 59 02](https://user-images.githubusercontent.com/90871/70802072-6ab24280-1da8-11ea-87e2-78def3feb89a.png)

-----

I think it seems fine to leave on one line, especially now that it's just a "Learn more" link visually.

Reply:
lol, never mind, I just got what I think you _actually_ meant, which is that the source code text is too long. I've made the change and removed the line break 😆